rpm/SPECS/docker-ce.spec

```diff
@@ -96,12 +96,16 @@ install -D -p -m 0755 /usr/local/bin/docker-init ${RPM_BUILD_ROOT}%{_bindir}/doc
 install -D -m 0644 engine/contrib/init/systemd/docker.service ${RPM_BUILD_ROOT}%{_unitdir}/docker.service
 install -D -m 0644 engine/contrib/init/systemd/docker.socket ${RPM_BUILD_ROOT}%{_unitdir}/docker.socket
 
+# create the config directory
+mkdir -p ${RPM_BUILD_ROOT}/etc/docker
+
```

Comment:
Seems like just `mkdir -p` in the `%install` section is the way to go, but if others know a more idiomatic approach, let me know 😅

Reply:
This is the correct idiom, and how you customize ownership/permissions :+1: